release_docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,159 @@
+        - Windows: [Testing HDF5 Binaries(this is missing)]() 
+    16. Manual Testing (i.e. verifying correct test outcomes via visual inspection): 
+        - Use this if UNIX test script is not reporting correct results, yet binaries look OK. 
+        - UNIX: [Manual Binary Testing of HDF5 on Unix systems (this is missing)]() 
+    17. Update the test results Confluence page with status/outcome of all test assignments.
+    18. If any test source (hdf-forum, clients, internal testers, automated regression suite) identifies any issues: 
+        - a) Enter the issue in JIRA summarizing the failure if it is not already there. 
+        - b) Decide whether or not to resolve the issue for this release, or to simply document it as a known issue for this release.
+        - c) Document and assign action items resulting from a) or b), and monitor them to closure.
+    19. Decide if another cycle of pre-release testing should occur based on the issue reports received and the actions taken during this cycle. If another round of testing is required (i.e. there were significant issues in pre-release testing which resulted in code changes), go back to step 7.2. If no further testing is required (i.e. no code changes were made and issues were documented as known issues, or code changes were trivial, unit tested, and exhaustive testing is unneeded), then proceed.
+
+
+8. Finalize Release Notes (Release Manager)
+    1. Perform a final review of release notes and ensure that any new changes made to the source, any new known issues discovered, and any additional tests run since the code freeze have been reflected in RELEASE.txt and other appropriate in-source documentation files (INSTALL_*, etc.). (Refer to the sub-steps of step 3 for what to check).
+    2. Update the [RELEASE.txt](https://github.com/HDFGroup/hdf5/blob/develop/release_docs/RELEASE.txt) in the **support** branch (i.e. hdf5_X_Y) to remove entries in “Bugs fixed” and “New Features” sections and increment the version number for the following release (“Bug fixes since X.Y.Z” - occurs twice).
+        - `$ git checkout hdf5_X_Y` 
+        - `$ vi RELEASE.txt # update RELEASE.txt to clear it out` 
+        - `$ git commit -m "Reset RELEASE.txt in preparation for the next release."` 
+        - `$ git push` 
+    3. Update Release Notes in **release** branch (Release Manager)
+
+9. Package and Distribute Release (Release Manager)
+    1. Run `bin/release` (similar to 8.2) and commit all the changed files.
+    2. Select release build from workflow.
+        - Choose the release branch
+        - Change ‘Release version tag’ name to 'hdf5_X.Y.Z' 
+        - Press "Run Workflow"
+    3. Review the release files in Github
+    4. Edit the Github Release and change status to Release
+        - Change status from Pre-release to Release
+
+10. Conduct Release Retrospective (Release Manager)
+    1. Schedule time and solicit comments from retrospective
+    2. Identify issues and document them
```

Comment:
Maybe a different numbering should be used for the sub-lists, for example:
1.
    a.
    b.
or
1.
    1.1
    1.2

Reply:
Ah, @byrnHDF already suggested about the numbering.

Reply:
Maybe make the tasks bold? Might help in distinguishing tasks from its steps?

Reply:
I tried changing how the formatting of that section looks. Please let me know if this is better.